packages/components/utils/src/Portal/Portal.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Portal'
+type: 'component'
+status: 'beta'
+slug: /utils/portal/
+typescript: ./Portal.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+`Portal` component can be used to render children in a DOM node outside of its parent component. It can be useful for rendering components that you don’t want to be clipped by their parent’s component. For example: modals, popovers, dropdown, etc.
+
+## Table of contents
+
+- [Import](#import)
+- [Examples](#examples)
+  - [Basic usage](#basic-usage)
+- [Props](#props)
+
+## Import
+
+```jsx static=true
+import { Portal } from '@contentful/f36-utils';
+```
+
+## Example
+
+### Basic usage
+
+```jsx static=true
+<Flex>
+  <Portal>
+    <Flex
+      style={{
+        backgroundColor: 'gray',
+      }}
+    >
+      Hi! I'm first in the document flow, but will be rendered at the end of the
+      document!
+    </Flex>
+  </Portal>
+  Parent component content
+</Flex>
+```
+
+## Props
```

Comment:
We now have a "Open in Storybook" button aligned with this title
and because of that we need to use some react components here

the [Accordion doc](https://github.com/contentful/forma-36/blob/forma-v4/packages/components/accordion/Accordion.mdx) has a good example of what needs to be done, but for the Portal page the code looks like this:

```suggestion
<Stack justifyContent="space-between" marginTop="spacing2Xl" marginBottom="spacingL">
  <Heading id="props-api-reference" as="h2" marginTop="none" marginBottom="none">
    Props (API reference)
  </Heading>

  <TextLink
    href="https://v4-f36-storybook.netlify.app/?path=/story/utilities-portal"
    target="_blank"
    alignIcon="end"
    icon={<ExternalLinkIcon />}
  >
    Open in Storybook
  </TextLink>
</Stack>
```

As you can see, it's necessary to import Stack, Heading, and TextLink on the top of the MDX file or this will fail

Reply:
but if there is no storybook page? should I add it?

Reply:
Good point!

I guess in that case, you can just do `## Props (API reference)` if they have any props